internal/cli/cmd/mod.go

```diff
@@ -22,12 +31,54 @@ func NewModCmd() *cobra.Command {
 	}
 
 	cmd.AddCommand(NewTidyCmd())
+	cmd.AddCommand(newModInitCmd())
 	cmd.AddCommand(newModDownloadCmd())
 	cmd.AddCommand(newModGetCmd())
 
 	return cmd
 }
 
+func newModInitCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "init [module-path]",
+		Short: "Initialize the module workspace with default values.",
+		Args:  cobra.MinimumNArgs(1),
+
+		RunE: fncobra.RunE(func(ctx context.Context, args []string) error {
+			dir, err := filepath.Abs(".")
+			if err != nil {
+				return err
+			}
+			if findroot.LookForFile(cuefrontend.WorkspaceFile, cuefrontend.LegacyWorkspaceFile)(dir) {
+				return fnerrors.New("workspace file aready exists.")
+			}
+			fmt.Println("Creating initial workspace.")
+			w := &schema.Workspace{
+				ModuleName: args[0],
+				EnvSpec:    cfg.DefaultWorkspaceEnvironmets,
+				Foundation: &schema.Workspace_FoundationRequirements{
+					MinimumApi: versions.MinimumAPIVersion,
+				},
+			}
+			mod, err := parsing.NewModule(ctx, dir, cuefrontend.WorkspaceFile, w)
```

Comment:
I'm wondering if this is overkill and we should just use a simple text template.

How fast is `ns mod init` right now? 
I'd expect it to be feel instant. If it does, let's roll with this approach.
Otherwise, let's speed it up and just use a text template :)